### DISCUSSION TITLE
Logs rotation

### DISCUSSION THREAD

**Discussion by AudreaM** [docker-library/rabbitmq#548] [Q&A]
Hello guys,

I've configured logs rotation inside my rabbitmq.conf.

```
# rotate every night at midnight
log.file.rotation.date = $D0

# keep up to 5 archived log files in addition to the current one
log.file.rotation.count = 5
```

But it seems that no rotation is made. Is this docker image does not manage this?

**Comment by tianon** [comment]
By default, logs for this image are managed by Docker, not RabbitMQ, so you'll want to adjust the configuration of your container/Docker log driver if you want to ensure your logs are rotating.

**Reply by AudreaM** [reply]
In my case I do not use Docker log driver, but I've got a volume mount on a directory '/var/log/rabbitmq'.
So logs are managed by RabbitMQ itself using env variable 'RABBITMQ_LOGS'.
So I was expecting RabbitMQ log rotation works as usual inside the container perhaps by using another env variable?
But if it's not possible, Ì'll try to find something else.

**Reply by tianon** [reply]
Ah, then you'll likely need to set/adjust the `log.file.rotation.*` values in your configuration file (see https://www.rabbitmq.com/logging.html#log-rotation).

**Reply by AudreaM** [reply]
Yeah, I already have these values inside my conf file as I said in my first message :).
But it doesn't work :(.
